src/main/java/io/supertokens/bulkimport/BulkImportUserUtils.java

```diff
@@ -83,7 +80,14 @@ private static List<String> getParsedUserRoles(JsonObject userData, List<String>
 
         // We already know that the jsonUserRoles is an array of non-empty strings, we will normalise each role now
         List<String> userRoles = new ArrayList<>();
-        jsonUserRoles.forEach(role -> userRoles.add(validateAndNormaliseUserRole(role.getAsString(), errors)));
+        jsonUserRoles.forEach(role -> {
+            String normalisedRole = validateAndNormaliseUserRole(role.getAsString(), errors);
```

Comment:
checking if a role exists should go in the validateAndNormaliseUserRole function right? Cause  the function has "validate" word in it

Reply:
Fixed